x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportForecastJobAction.java

```diff
@@ -140,5 +175,14 @@ static void validate(Job job, ForecastJobAction.Request request) {
                                 + duration.getStringRep() + "/" + bucketSpan.getStringRep() + "]");
             }
         }
+
+        if (request.getMaxModelMemory() != null) {
+            if (request.getMaxModelMemory() >= FORECAST_LOCAL_STORAGE_LIMIT.getBytes()) {
+                throw ExceptionsHelper.badRequestException(
+                    "[{}] must be less than 500MB",
```

Comment:
nit: you used the constant in the if, so it makes sense to use it here, too

Reply:
@hendrikmuhs possibly? I am not sure if the `stringRep` for this value will be 500mb. I just wanted to guarantee a readable error message.